File: IronVault/UI/ironvault_desktop/lib/models/nutricionist_seminar.dart
import 'package:ironvault_desktop/models/seminar.dart';
import 'package:json_annotation/json_annotation.dart';

part 'nutricionist_seminar.g.dart';

@JsonSerializable()
class NutricionistSeminar {
  int? nutricionistSeminarId;
  int? nutricionistId;
  int? seminarId;
  Seminar? seminar;

  NutricionistSeminar({this.nutricionistSeminarId});

  factory NutricionistSeminar.fromJson(Map<String, dynamic> json) =>
      _$NutricionistSeminarFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$NutricionistSeminarToJson(this);
}

File: IronVault/IronVault.Model/Models/NutricionistSeminar.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class NutricionistSeminar
    {
        public int NutricionistSeminarId { get; set; }

        public int NutricionistId { get; set; }

        public int SeminarId { get; set; }

        public Seminar Seminar { get; set; }
    }
}
